modules/core/arrow-core-data/build.gradle

```diff
@@ -10,9 +10,8 @@ dependencies {
     testRuntime("org.junit.vintage:junit-vintage-engine:$jUnitVintageVersion")
     testCompile "io.kotlintest:kotlintest-runner-junit4:$kotlinTestVersion"
     testCompile project(':arrow-test')
-    testCompile project(':arrow-syntax')
+    testCompile project(':arrow-syntax') // TODO: remove
```

Comment:
What is the reason behind this? If this remains here, it'll bloat the binary by almost a megabyte.